skbio/alignment/_tabular_msa.py

```diff
@@ -1509,8 +1626,7 @@ def _get_join_index(self, other, how):
 
     def _get_sequence_for_join(self, label):
         if label in self.index:
-            # TODO: use .loc when it is implemented.
```

Comment:
Strangely, after looking through all of the TODO/todo in our package, none of them came up mentioning loc/iloc/getitem